people_tool/templates/people_tool/partials/search.html

```diff
@@ -0,0 +1,94 @@
+{% verbatim %}
+<nav>
+  <h3 class="breadcrumbs">
+    {% endverbatim %}
+    <a href="{% url 'dashboard_account' %}">Admin Console</a>
+    {% verbatim %}
+    <small><i class="fa fa-chevron-right"></i></small>
```

Comment:
More of a note to ourselves and @Vittorio2015 : Here and Crosslist tool, we now use the <small><i class="fa fa-chevron-right"></i></small>, but in the Course info(Course search) , we use the '>' symbol, which also has a different style. We need to modify that at some point to be consistent.
https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/e1179ec4ebc443417223327bb6bf3d1646eb0bde/course_info/templates/course_info/partials/search.html#L5